src/common/mipmap_cache.c

```diff
@@ -77,11 +79,12 @@ struct dt_mipmap_buffer_dsc
   uint32_t pre_monochrome_demosaiced;
   dt_colorspaces_color_profile_type_t color_space;
   /* NB: sizeof must be a multiple of 4*sizeof(float) */
-} __attribute__((packed, aligned(16)));
+} __attribute__((packed, aligned(4 * sizeof(float))));
```

Comment:
Hmm, yeah, i think i may have to factor out all alignments into some define and use it everywhere.
But this particular change just seemed more readable at the moment of commit, i'll drop it..